apps/chat/src/components/Chat/Publish/PublishWizard.tsx

```diff
@@ -331,15 +331,15 @@ export function PublishModal({
     <Modal
       portalId="theme-main"
       containerClassName={classNames(
-        'group/modal  inline-block h-[747px] min-w-full max-w-[1100px] !bg-layer-2 md:min-w-[550px] lg:min-w-[1000px] xl:w-[1100px]',
+        'group/modal flex min-h-[579px] min-w-full max-w-[1100px] !bg-layer-2 md:h-[747px] md:min-w-[550px] lg:min-w-[1000px] xl:w-[1100px]',
         { 'w-full': files.length },
       )}
       dataQa="publish-modal"
       state={isOpen ? ModalState.OPENED : ModalState.CLOSED}
       onClose={onClose}
       initialFocus={nameInputRef}
     >
-      <div className="flex h-full flex-col divide-y divide-tertiary">
+      <div className="flex flex-col divide-y divide-tertiary overflow-auto">
```

Comment:
I don't think that we want to allow horizontal scroll
```suggestion
      <div className="flex flex-col divide-y divide-tertiary overflow-y-auto">
```